components/alert/alert.css

```diff
@@ -0,0 +1,71 @@
+@import url("/docs/main.css");
+
+.alert_container {
+    background-color: var(--white);
+    display: flex;
+    flex-direction: column;
+    min-width: fit-content;
+    margin: 2rem auto;    
+    padding: 1.5rem 0rem;
+    justify-content: flex-start;
+    border-bottom: 1px solid var(--line);
+    flex-wrap: wrap;
+    overflow: auto;
+    text-align: left;
+}
+
+.alert_head {
+    font-size: 1.5rem;
+    padding-top: 2rem;
+    border-top: 1px solid var(--line);
+}
+
+.alert_desc {
+    margin: 2rem 0;
+    font-size: 1rem;
+    color: var(--light-grey);
+    font-weight: 400;
+}
+
+.alert_box {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+}
+
+.alert_success {
+    display: flex;
+    margin: 1rem;
+    padding: 1rem;
+    color: #1b7052;
+    background-color: #84f1c9;
```

Comment:
Maybe make variables for these colors as you will be using them again.

Reply:
Thanks for reminding, Mahesh. I foreget this. Made changes now.